modules/core/src/v2/wallet.ts

```diff
@@ -1765,9 +1765,24 @@ export class Wallet {
         throw new Error('txPrebuild must be an object');
       }
       const presign = yield self.baseCoin.presignTransaction(params);
-      const userPrv = self.getUserPrv(presign);
-      const signingParams = _.extend({}, presign, { txPrebuild: txPrebuild, prv: userPrv });
-      return self.baseCoin.signTransaction(signingParams);
+      const externalSignerUrl = this.bitgo._externalSignerUrl;
+      if (externalSignerUrl) {
+        return fetch(externalSignerUrl, {
```

Comment:
please don't mix promise syntax, let's stick with `yield` for now instead of using `.then()`

Reply:
also same comment as later on about using fetch vs superagent.